lib/ansible/modules/cloud/amazon/sqs_queue.py

```diff
@@ -147,19 +201,20 @@
 import traceback
 
 try:
-    import boto.sqs
-    from boto.exception import BotoServerError, NoAuthHandlerFound
-    HAS_BOTO = True
-
+    import botocore
+    import boto3
+    HAS_BOTO3 = True
```

Comment:
Not required with AnsibleAWSModule

Reply:
Was basing this off another module - any documentation I can see that will help me understand this better?